### DISCUSSION TITLE
ダークモードをサポートしてほしい

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#103] [要望・アイデア・不具合]
近年、ダークモードをサポートしているウェブサイトが増えてきています（エンジニアにとってはGitHubが身近な例でしょう）。[^1]
文章を読むのが主体であるQiitaでもダークモードをサポートしていただけると、目に優しくなっていいかと思います。

僕自身は、[Dark Reader](https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh)というChrome拡張で勝手にダークモード化しています。

|通常|ダークモード化|
|---|---|
|<img width="1268" alt="スクリーンショット 2021-12-30 13 40 32" src="https://user-images.githubusercontent.com/20086673/147722227-ebf136ca-1a81-409f-90e9-6978798fb7e7.png">|<img width="1268" alt="スクリーンショット 2021-12-30 13 42 48" src="https://user-images.githubusercontent.com/20086673/147722338-7e2b6d9a-8b6f-4cf0-a69e-6bb27582909e.png">|

ただしこれは機械的にスタイルをあてているだけなので、場合によっては見づらくなるケースもあります。
公式でちゃんとデザインされたダークモードがあればそれを使うにこしたことはないので、ダークモードをサポートしていただけるとありがたいです。

[^1]: https://github.com/munierujp/awesome-dark-mode

**Comment by kskwtnk** [comment]
@munierujp 
ご意見ありがとうございます！

こちらは非常に多くの方から要望がありますし、私個人としても実施したいのですが「今のコードでは実施ができない」が回答です 🙇 

詳細は省略させていただきたいのですが、今実施しようと思うと全ての要素にメディアクエリを書いて`prefers-color-scheme`を指定する以外の術が無い……ようなイメージです

実施したい気持ちは非常に強いのですが、積み上がった過去のコードをどうにかしないと実現できないため、目下そちらを整理している状況です
ご理解いただけますと幸いです 🙇

**Comment by kskwtnk** [comment]
@munierujp 

本日、ベータ版としてダークモードを公開しました！

https://blog.qiita.com/article-page-dark-mode-beta/

現在は記事ページだけですが、今後も対象を拡充し、正式版に向けて進めていく予定です！
道半ばではありますが共有でした！

**Reply by munierujp** [reply]
ありがとうございます！ダークモードとてもいいですね🥰
